e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+create_infra "$namespace"
+cluster="some-name"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running "$cluster-rs0" 3
+
+check_service() {
+	kubectl_bin patch psmdb "$cluster" --type=json --patch '[
+        {
+            "op": "replace",
+            "path": "/spec/ignoreAnnotations",
+            "value": ["ignoredAnnotation"]
+        },
+        {
+            "op": "replace",
+            "path": "/spec/ignoreLabels",
+            "value": ["ignoredLabel"]
+        }
+    ]'
+
+	sleep 10
+
+	replset_name=$1
+
+	local expose_path
+	local svc_name
+	case $replset_name in
+
```

Comment:
Why we can't create the cluster with exposed services? We anyway use custom CR from test dir.

Reply:
Services are already exposed in the custom cr